src/components/Notifications/NoticeBoard.tsx

```diff
@@ -19,24 +20,39 @@ export const NoticeBoard = () => {
 
   if (data?.results.length) {
     notices = (
-      <div className="grid grid-cols-1 gap-6 sm:grid-cols-2 xl:grid-cols-3">
+      <div className="grid grid-cols-1 gap-6 sm:grid-cols-2 xl:grid-cols-3 mt-4">
         {data.results.map((item) => (
           <div
             key={`usr_${item.id}`}
-            className="overflow-hidden rounded shadow-md"
+            className="overflow-hidden rounded shadow-md my-1"
           >
-            <div className="px-6 py-4">
-              <div className="text-justify text-lg" id="notification-message">
-                {item.message}
-              </div>
-              <div className="text-md my-2 text-secondary-700">
-                {formatName(item.caused_by)} -{" "}
-                <span className="font-bold text-primary-700">
-                  {item.caused_by.user_type}
-                </span>
-              </div>
-              <div className="text-xs text-secondary-900">
-                {t("on")}: {formatDateTime(item.created_date)}
+            <div
+              className="text-justify text-lg flex mx-2 py-3 px-5"
+              id="notification-message"
+            >
+              <CareIcon
+                icon="l-facebook-messenger"
+                className="text-xl mr-6 mt-1"
+              />
+              <span className="font-mono text-md">{item.message} </span>
+            </div>
+
+            <div className="bg-gray-200 py-2 flex items-center ">
+              <Avatar
+                name={item.caused_by.user_type || ""}
```

Comment:
Missing the image url in Avatar.

Reply:
@Jacobjeevan Added the missing image url in Avatar